### PR TITLE
fix(dp): Fix domain-proxy image build job notifications

### PR DESCRIPTION
**Pull request by sklgromek**
## Summary
Domain proxy image build CI  job was reporting success on failure, because of mistaken conditional.

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Review by Neudrino — APPROVED** [review]

**Comment by xbend** [comment]
@magma/approvers-ci 
Ready to merge and waiting for your approval.

**Review by talkhasib — APPROVED** [review]

**Review by 119Vik — APPROVED** [review]

**Pull request merged**
